### DISCUSSION TITLE
Is it possible to skip all current major version PRs when enabling the repo?

### DISCUSSION THREAD

**Discussion by matthewnessworthy** [renovatebot/renovate#16706] [Request Help] [answered]
For some older repositories I would like to skip all the initial pull requests and only have new versions after enabling the repo show up. Is this possible?

For reference, 1 repo would create 70 Pull Requests with the configuration I have selected.

**Comment by viceice** [comment]
With default `config:base` renovate will only raise 2 PR per hour.

You should enable dashboard approval, so you can manual approve the PR's.

https://docs.renovatebot.com/getting-started/use-cases/#dependency-dashboard-approval

**Comment by rarkins** [comment, marked as answer]
If you are asking "Can you upgrade to new versions only if they are released after the date I onboarded the repo" then the answer is no. Like @viceice said, I suggest you set `dependencyDashboardApproval=true` for all dependencies so that you get no PRs without manually approving them first.
